documentation-website/cypress/e2e/contributors.cy.ts

```diff
@@ -12,7 +12,8 @@ describe('contributors', () => {
   },);
   it('title matches', () => {
     cy.title()
-      .should('eq', 'Contributors | Contributing | @idrinth/api-bench',);
+      .should('eq', 'Contributors | Contributing | '+
```

Comment:
this package will never exist, please revert the change for now

Reply:
Reverted